frontend/mgramseva/lib/screeens/ConsumerDetails/ConsumerDetails.dart

```diff
@@ -287,7 +287,7 @@ class _ConsumerDetailsState extends State<ConsumerDetails> {
                                   textInputType: TextInputType.number,
```

Comment:
Input type should not be TextInputType.numer

Reply:
As we are using input Formatter,  So text input type is of no use, So I have removed it now